docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/listing/list_item_macros.twig

```diff
@@ -339,3 +339,66 @@
     </div>
   </li>
 {% endmacro %}
+
+{% macro ncids_dynamic_list_event_macro(title, startDate, endDate, description = '', url, series = '', allDay = '', venue = '', room = '', cityState = '') %}
+  {% set eventStart = startDate[0]['#attributes']["datetime"] %}
+  {% set eventEnd = endDate[0]['#attributes']["datetime"] %}
+  {% set isMultiDayEvent = eventStart|date('Y-m-d') < eventEnd|date('Y-m-d') %}
+  {% set isMultiYearEvent = eventStart|date('Y') <  eventEnd|date('Y') %}
+  <li class="usa-collection__item">
+    <div class="usa-collection__body">
+      {# Title #}
+      <h3 class="usa-collection__heading">
+        <a class="usa-link" href="{{ url }}">
+          {{ title }}
+        </a>
+      </h3>
+      <ul class="usa-collection__meta" aria-label="testing label">
+        <li class="usa-collection__meta-item">
+          {# Series name #}
+          {% if series|field_value %}
+            <strong>{{ series }}</strong>
+            <br />
+          {% endif %}
+          {# Date/time range #}
+          {% if isMultiDayEvent %}
+            {% if isMultiYearEvent %}
+              {{ eventStart|date('F j, Y') }}
+              &ndash;
+              {{ eventEnd|date('F j, Y') }}
+            {% else %}
+              {{ eventStart|date('F j') }}
+              &ndash;
+              {{ eventEnd|date('F j, Y') }}
+            {% endif %}
+          {% else %}
+            {{ eventStart|date('F j, Y') }}
+            {% if allDay != "On" %}
+              &vert;
+              {{ eventStart|date('g:i A') }}
+              &ndash;
+              {{ eventEnd|date('g:i A') }}
+            {% endif %}
+          {% endif %}
```

Comment:
These should probably be `<time>` elements in keeping with our guidance on using dates and times as `<time>` elements. 

Also, there is no real context here -- this is something schema.org has a schema for, so adding context is probably for the future.

Reply:
We should skip time for now. Really we should make sure the date is honoring timezones and whatnot for this to be useful for computers.